### PR TITLE
Add Chinese translation of mixin-class-composition

### PR DESCRIPTION
**Pull request by coalchan**

### CODE REVIEW

**Comment by coalchan** [comment]
@xingyif if you get time, can you help me to review it.

File: _zh-cn/tour/mixin-class-composition.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: tour
-title: Class Composition with Mixins
+title: 通过混入来组合类
```
Comment (xingyif) [inline _zh-cn/tour/mixin-class-composition.md]: I think we should put (mixin) next to `混入`.

**Comment by coalchan** [comment]
Conversely, English is difficult for me but also interesting in programming. :smile:

**Comment by SethTisue** [comment]
thank you @coalchan and @xingyif !

**Pull request merged**